src/lib/rc/srxl.cpp

```diff
@@ -0,0 +1,488 @@
+#include <cstring>
```

Comment:
copyright header

Reply:
Meaning you would like to see the same copyright header in every file?